.travis.yml

```diff
@@ -10,8 +10,7 @@ cache:
  directories:
   - $HOME/.composer/cache
 
-before_script:
-  - travis_retry composer self-update
```

Comment:
I've removed this as `composer` itself is being updated on Travis already, see for example https://travis-ci.org/bmitch/churn-php/jobs/485124364#L474-L476.